cmd/server/assets/codestatus/index.html

```diff
@@ -21,15 +22,16 @@ <h1>Verification code status</h1>
     </p>
 
     <div class="card mb-3">
+      <div class="card-header">Code status</div>
       <div class="card-body">
-        <form method="POST" action="show">
+        <form method="POST" action="show" class="floating-form">
           {{ .csrfField }}
-          <div class="row form-group">
-            <label for="uuid" class="col-sm-3">Code identifier</label>
-            <div class="col-sm-9">
+          <div class="form-group">
+            <div class="form-label-group ">
               <input type="text" id="uuid" name="uuid" class="form-control text-monospace{{if $code.ErrorsFor "uuid"}} is-invalid{{end}}" value="{{$code.UUID}}"
                 placeholder="xxxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx"
                 autocomplete="off">
+                <label for="uuid">xxxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx</label>
```

Comment:
I think we want the placeholder to be XXXXXX..., but the floating label should be "UUID" or something

Reply:
I think the two should be the same (whatever that is). It starts with placeholder text and smoothly moves up as you type. Ideally it would be UUID and we'd have some AsYouType formatting going on

Reply:
Ah I see. I missed that it was an animation.